pkg/jwt/bearer_token.go

```diff
@@ -0,0 +1,54 @@
+package jwt
```

Comment:
Please rename. This is clashes with `"github.com/golang-jwt/jwt/v5"`, and is confusing.

Reply:
changed to `authorization`